### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by M-a-c** [swdotcom/swdc-vscode#302]
<img width="1674" alt="Screen Shot 2021-07-08 at 6 43 03 PM" src="https://user-images.githubusercontent.com/1916685/125003605-2f780400-e01d-11eb-9db6-236dc964b930.png">
**Describe the bug**
```paste below
High contrast mode in Visual Studio code breaks the text in the side bar.
```
**ISSUE TYPE**
- Bug Report

**Steps to reproduce**

1. Open visual studio code
2. Preferences > Themes > Workbench: Color Theme
3. Select "High Contrast"

**Expected behavior**
```paste below
I expect the text to be white. or some other high visibility color.
```

**ACTUAL RESULTS**

```paste below
Text is not visible.
```

**Screenshots**

<img width="1674" alt="Screen Shot 2021-07-08 at 6 43 03 PM" src="https://user-images.githubusercontent.com/1916685/125003605-2f780400-e01d-11eb-9db6-236dc964b930.png">
**Desktop (please complete the following information):**
 - OS: 11.2 OSX
 - Plugin version: Code Time v2.5.16

**Additional context**
```paste below

```

**Comment by xavluiz** [comment]
Thank you for the issue update. We'll look into this right away.

**Comment by xavluiz** [comment]
We published a fix this this in v2.5.19. Thank you.
